### PR TITLE
2 bz ord ml bd 2

### LINKED ISSUE #2: Multilayer Graphs
Create:
- [ ] _MultilayerWD.gen.m (Blanca)
- [x] #12 (Hang)
- [x] _MultilayerBD.gen.m (Blanca)
- [x] #19

Create:
- [x] _OrdMlWD.gen.m (Hang)
- [x] _OrdMlWU.gen.m (Hang)
- [ ] _OrdMlBD.gen.m (Blanca)
- [ ] _OrdMlBU.gen.m (jws)

Create
- [x] _OrdMxBUD.gen.m (Hang)
- [x] _OrdMxBUT.gen.m (Blanca)
- [x] #11 (jws)
- [x] _MultilayerBUT.gen.m (Blanca)
- [ ] _OrdMlBUD.gen.m (jws)
- [ ] _OrdMlBUT.gen.m (jws)

### PR DESCRIPTION
**Pull request by bzufiriaki**
Added:

- new OrdMlBD graph

- modify Distance measure for OrdMlBD

### CODE REVIEW

File: braph2genesis/graphs/_MultilayerWD.gen.m
Comment (hanzh346) [inline braph2genesis/graphs/_MultilayerWD.gen.m:1]: IN the description, should it be the _all connections are allowed in layers_?

File: braph2genesis/graphs/_OrdMlBD.gen.m [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.NONNEGATIVE * ones(layernumber);
+
+%%% ¡prop!
+A (result, cell) is the cell containing the within-layer binary adjacency matrices of the multilayer binary directed graph and the ordinal connections between layers..
```
Comment (annacanal) [inline braph2genesis/graphs/_OrdMlBD.gen.m]: you have two final dots (.)

File: braph2genesis/graphs/_OrdMlBD.gen.m [hunk 2]
```diff
@@ -0,0 +1,274 @@
+%% ¡header!
+OrdMlBD < Graph (g, ordinal multilayer binary directed graph) is a ordinal multilayer binary directed graph.
+
+%%% ¡description!
+In an ordinal multilayer binary directed (BD) graph, layers could have the different number 
+ of nodes with within-layer directed edges. Edges can be either 0 (absence of connection) 
+or 1 (existence of connection).
+All node connections are allowed between layers.
+The layers are connected in an ordinal fashion, i.e., only consecutive layers are connected.
```
Comment (annacanal) [inline braph2genesis/graphs/_OrdMlBD.gen.m]: ... layers could have different number 
 of nodes with within-layer directed edges. (remove THE before different)

File: braph2genesis/graphs/_OrdMlBD.gen.m [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+
+%%% ¡prop!
+NAME (constant, string) is the name of the ordinal multilayer binary directed graph.
+%%%% ¡default!
+'OrdMlBD'
+
+%%% ¡prop!
+DESCRIPTION (constant, string) is the description of the ordinal multilayer binary directed graph.
+%%%% ¡default!
+'In an ordinal multilayer binary directed (BD) graph, layers could have the different number  of nodes with within-layer directed edges. Edges can be either 0 (absence of connection)  or 1 (existence of connection). All node connections are allowed between layers. The layers are connected in an ordinal fashion, i.e., only consecutive layers are connected.'
```
Comment (annacanal) [inline braph2genesis/graphs/_OrdMlBD.gen.m]: ... layers could have different number 
 of nodes with within-layer directed edges. (remove THE before different)

**Review by annacanal — CHANGES_REQUESTED** [review]

**Review by annacanal — APPROVED** [review]

File: braph2genesis/graphs/_OrdMlBD.gen.m [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+%%% ¡test!
+%%%% ¡name!
+Constructor - Full
+%%%% ¡probability!
+.01
+%%%% ¡code!
+B1 = rand(randi(10));
+B2 = rand(randi(10));
+B3 = rand(randi(10));
+B12 = rand(size(B1,1),size(B2,2));
```
Comment (JiaweiSun0512) [inline braph2genesis/graphs/_OrdMlBD.gen.m:183]: need space after ','

**Review by hanzh346 — APPROVED** [review]

**Pull request merged**